PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle

```diff
@@ -1,26 +1,7 @@
-buildscript {
-    ext.kotlin_version = '1.2.10'
-
-    repositories {
-        mavenCentral()
-        maven {
-            url "https://oss.sonatype.org/content/repositories/snapshots/"
-        }
-        maven {
-            url 'http://dl.bintray.com/jetbrains/intellij-plugin-service'
-        }
-    }
-
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
-}
-
 plugins {
-    id "org.jetbrains.intellij" version "0.3.3"
+    id "org.jetbrains.intellij" version "0.3.4"
+    id "org.jetbrains.kotlin.jvm" version "1.2.50"
```

Comment:
Can we use
```gradle
plugins {
    kotlin("jvm") version "1.2.50"
}
```
?

Reply:
Yes we can, but from the docs, I don't see any difference between the 2 forms. Just wondering, do you know the differences between the `Gradle plugins DSL` and `Gradle Kotlin DSL`?

Reply:
Aha, Gradle Kotlin DSL looks more aggressive, it 's 

>  a Kotlin-based approach to writing Gradle build scripts and plugins

Let's just keep the current version, groovy is enough currently.

Reply:
Ok, it makes sense.